backend/src/main/java/com/crm/dto/request/NewAccountRequest.java

```diff
@@ -0,0 +1,53 @@
+package com.crm.dto.request;
+
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.ReadOnlyProperty;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Getter
+public class NewAccountRequest {
+
+    @NotNull(message = "Nazwa użytkownika nie może być pusta")
```

Comment:
Messages are repeating in another class. You can extract error messages to an ErrorDict class to variables

Reply:
Let's keep messages in English ;)